plugins/board-resources/src/components/popups/EditMembers.svelte

```diff
@@ -0,0 +1,97 @@
+<script lang="ts">
+  import type { Card } from '@anticrm/board'
+  import { createQuery } from '@anticrm/presentation'
+  import type { Ref, TxOperations } from '@anticrm/core'
+  import contact, { Employee } from '@anticrm/contact'
+  import { Label, Scroller, SearchEdit, ActionIcon, IconClose } from '@anticrm/ui'
+  import { DocumentQuery } from '@anticrm/core'
+  import board from '../../plugin'
+  import Member from './Member.svelte'
+
+  export let object: Card
+  export let client: TxOperations
+  export let onClose: () => void
+
+  const query = createQuery()
+
+  let search = ''
+  let resultQuery: DocumentQuery<Employee> = {}
+
+  $: resultQuery = search === '' ? {} : { $search: search }
```

Comment:
Better to use name: "%pattern%" for like search.

Reply:
Done.